src/librustdoc/html/templates/item_info.html

```diff
@@ -0,0 +1,7 @@
+{%- if !items.is_empty() -%}
```

Comment:
```suggestion
{% if !items.is_empty() %}
```

And the same for all `-`.

Reply:
Should the same be done for #108784 in that case?

Reply:
Yes indeed. It's because I changed the default whitespace handling to "suppress" in askama config.